content/blog/2020/05/2020-05-18-read-only-jenkins-announcement.adoc

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Read only jenkins announcement"
```

Comment:
```suggestion
title: "Read-only Jenkins Configuration"
```
It seems none of the other announcements use that word in the title https://www.jenkins.io/node/tags/announcement/